lib/workflows/vm-migration/begin.js

```diff
@@ -187,6 +187,9 @@ function createProvisionPayload(job, cb) {
 
 
     var vmPayload = job.vmPayload = clone(job.params.vm);
+    var brand = vmPayload.brand;
+
+    assert.string(brand, 'brand');
```

Comment:
This is gonna be run by workflow, if this throws an exception, you'll get no error message for the job results. I'd say we shouldn't use assert here and change to the approach below where you're returning an explicit error message when there's more than one dataset.

Reply:
Yep - good point - will change to a callback error.